pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -61,6 +61,11 @@ struct ct_entry {
      */
     __u8  flags_seen_tx_dir;
     __u8  flags_seen_rx_dir;
+
+    /**
+     * last_seq stores the last sequence number seen in the connection.
+     */
+    __u32 last_seq;
```

Comment:
nit: might check if there is another place in this struct to put this so it is better optimized i.e. less paddings